packages/docs/docs/guide/deploy.md

```diff
@@ -180,3 +180,39 @@ git push heroku master
 # opens a browser to view the Dashboard version of Heroku CI
 heroku open
 ```
+
+## Now
+
+1. Install the Now CLI globally: `npm install -g now`
+
+2. Add a `docs.now.json` file to your project root:
+
+    ```json
```

Comment:
Remove the indent so the code fence can display better at mobile side.

Reply:
you mean the indentation inside the json template string for the docs.now.json?